### PR TITLE
Mention command pallet issue in Paeredit guide.

### PR DESCRIPTION
**Pull request by ieugen**

### CODE REVIEW

**Comment by PEZ** [comment]
Hello, thanks for this PR!

It is only a few of the commands that are affected, and as far as the guide goes, maybe it is just the **forward/backward sexp** ones. I think it is better to just spell out that for Mac it is `alt+right/left` and for Windows and Linux it is `ctrl+ right/left`  when navigation is introduced, rather than a general disclaimer.

File: drams/calva_getting_started/hello_paredit.clj [hunk 1]
```diff
@@ -62,6 +62,9 @@
 ;; Move form-by-form using *Paredit Sexp Forward*
 ;; and *Paredit Sexp Backward*
 
+;; Note: For Mac it is alt+right/left and
+;; for Windows and Linux it is ctrl+ right/left
+
```
Comment (PEZ) [inline drams/calva_getting_started/hello_paredit.clj]: I think it was a good idea you had, to mention the issue.

```suggestion
;; Note: Despite what the command palette is showing, the
;; Keyboard shortcuts for For Mac are alt+right/left and
;; for Windows and Linux they are ctrl+right/left
;; See https://github.com/BetterThanTomorrow/calva/issues/1161
```

**Review by PEZ — APPROVED** [review]
Thanks. What do you think about the suggestion I added?

File: drams/calva_getting_started/hello_paredit.clj (hunk 1 above)
Comment (ieugen) [reply drams/calva_getting_started/hello_paredit.clj]: kewl

**Comment by PEZ** [comment]
Thanks for helping with this!

**Comment by ieugen** [comment]
Thank you for making such a great piece of software and for looking into my issue.
I'll try to contribute more over time.

**Pull request merged**